CDP4Dal/CDPMessageBus.cs

```diff
@@ -68,13 +68,40 @@ public static CDPMessageBus Current
         /// Total number of Calls that have been made to the Listen method
         /// during the lifetime of this instance of <see cref="CDPMessageBus"/>.
         /// </summary>
-        public int ListenerCallCount { get; private set; } 
+        public int ListenerCallCount { get; private set; }
+
+        /// <summary>
+        /// Backing field for the <see cref="IsActive" />
+        /// </summary>
+        private bool isActive;
+
+        /// <summary>
+        /// Flag indicates that the messages bus notifications are enabled or not
+        /// </summary>
+        public bool IsActive => this.isActive;

```

Comment:
i propose we rename this to "IsEnabled" to be inline with the Enable/Disable methods. if you do rename, make sure the backing field on line 76 is renamed as well/

Please rephrase docs using the COMET style for boolean:
```
Gets a value indicating whether the <see cref="CDPMessageBus"/> is active or not
```

Also curious why you choose to have a backing field, a read-only property and 2 methods to set the backing field. Why not use a property with a getter and a setter?